bundles/org.openhab.binding.airq/src/main/java/org/openhab/binding/airq/internal/AirqHandler.java

```diff
@@ -460,11 +427,14 @@ public int getResponseCode() {
 
     @Override
     public void dispose() {
-        if (pollingJob != null) {
-            pollingJob.cancel(true);
+        ScheduledFuture<?> localPollingJob = pollingJob;
+        if (localPollingJob != null) {
+            localPollingJob.cancel(true);
```

Comment:
Don't you also need to set pollingJob = null same for getConfigDataJob; down below?

Reply:
It won't harm, but it also has no benefit, or do I miss something? I saw this pattern in other bindings, but I never understood the purpose...

Reply:
Same here, so we have the same questions :-)